pipeline-steps/deploy.groovy

```diff
@@ -0,0 +1,19 @@
+def deploy(){
+  common.conditionalStep(
+    step_name: "Deploy",
+    step: {
```

Comment:
Heh, I should have checked earlier, that method is much easier, can use your common.run_script. However this method will be good for customisation, re-running etc.